docs/scalewaycluster.md

```diff
@@ -192,6 +189,26 @@ spec:
 > When `private` is set to `true`, make sure your management cluster has network access
 > to the Private Network where the workload cluster will be created.
 
+The main Load Balancer's port can be set at the cluster creation in the `Cluster` object.
+
+Here is an example of the main Load Balancer port configuration:
+
+```yaml
+apiVersion: cluster.x-k8s.io/v1beta1
+kind: Cluster
+metadata:
+  name: my-cluster
+  namespace: default
+spec:
+  ...
+  clusterNetwork:
+    apiServerPort: 443
+  ...
+```
+
+- The `apiServerPort` field specifies the port of the Load Balancer frontend that exposes the kube-apiserver(s).
+  This field is immutable.
```

Comment:
Isn't this field actually mutable (i.e. the Kubernetes API does not prevent updating this value)?
I think we should just recommend not to change it.